typescript/sdk/src/hook/EvmHookReader.ts

```diff
@@ -156,11 +152,7 @@ export class EvmHookReader implements HookReader {
     const hookConfigs: DerivedHookConfig[] = await concurrentMap(
       this.concurrency,
       hooks,
-      async (hook) => {
-        const hookConfig = await this.deriveHookConfig(hook);
-        assert(hookConfig, `No hook config found for ${hook}.`);
-        return hookConfig;
-      },
+      async (hook) => this.deriveHookConfig(hook),
```

Comment:
nit: is this the same?
```suggestion
      this.deriveHookConfig,
```

Reply:
one passes in just a method, one passes in the method and the necessary context to make it not error